marketing_security_group/wizards/test_mailing.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3: Antiun Ingenieria S.L. - Antonio Espinosa
+# See README.rst file on addon root folder for more details
+
+from openerp import models, fields
+
+
+class MailMassMailingTest(models.Model):
+    _inherit = 'mail.mass_mailing.test'
+
+    mass_mailing_id = fields.Many2one(ondelete='cascade')
```

Comment:
This is because if user has sent a test email, then a ```mail.mass_mailing.test``` temporal record is created and before vacuuming it, removing ```mail.mass_mailing``` is throwing a n ORM exception because there is object referencing to it.
 
Actually, I've inherited this model using ```models.Model```, I've already fixed this.